packages/synapse-react-client/src/components/FileUpload/FileUpload.tsx

```diff
@@ -76,6 +76,7 @@ export const FileUpload: React.FC<FileUploadProps> = props => {
   return (
     <>
       <input
+        data-testid="file-input"
```

Comment:
Added to more easily mock DUC template upload in tests for `SetManagedAccessRequirementFields`